prov/verbs/src/verbs_msg_ep.c

```diff
@@ -238,7 +215,13 @@ static int fi_ibv_msg_ep_enable(struct fid_ep *ep)
 	}
 
 	attr.qp_type = IBV_QPT_RC;
-	attr.sq_sig_all = 0;
+
+	/* assuming FI_EP_SOCK_STREAM */
```

Comment:
Add an assert() or validation check

Reply:
^ +1

The EP type should be checked and further checks can be done based on that. If needed, the checks can be put in a separate function. We should not remove the existing checks for msg EP type.